src/components/shared/layouts/pageWrapper.tsx

```diff
@@ -12,6 +13,7 @@ type Props = {
 const PageWrapper = ({ children, className = '' }: Props) => (
   <div className={styles.page}>
     <GlobalToolbar />
+    <Alert />
```

Comment:
Just found a "bug" (not really a bug, moreso weird behavior). This component rerenders with page changes, which is probably not quite what we want. 

The way to fix this is to move this up in the component chain. Probably into `app.tsx` since there's not really much that's above the page wrapper (there's the router, but I wouldn't want this there).

If we do that, we're also gonna want to up our z-index of our loading overlay to 2 since on page transition we wouldn't want this popup to be seen above that overlay

Reply:
i moved it and it still re-renders on page change

Reply:
No it doesn't what you did is fine.